__tests__/app.test.js

```diff
@@ -107,3 +107,36 @@ describe("* GET/api/articles", () => {
       });
   });
 });
+
+describe("* GET/api/articles/:article_id", () => {
+  test("returns status code 200", () => {
+    return request(app).get("/api/articles/3").expect(200);
+  });
+
+  test("responds with a single matching article", () => {
+    return request(app).get("/api/articles/3").expect(200)
+    .then(({ body }) => {
+      expect(body).toEqual(
```

Comment:
You should be sending data back on the key of 'article' e.g.
```
{ body: article : { article_id: 3 .. } } 
```

Reply:
this has been refactored, thank you :)